AE_PunktBeschriftung/include/representations/Solution.hpp

```diff
@@ -1,32 +1,58 @@
-#pragma once
+#ifndef LABEL_MAP_SOLUTION_HPP
+#define LABEL_MAP_SOLUTION_HPP
 
-#include <representations/Instance.hpp>
-#include <representations/Point.hpp>
 
-#include <map>
+#include "Instance.hpp"
```

Comment:
We have a line in the CMakeLists.txt that adds the directory "include" to the include-directories, so that we can use <> here

Reply:
more stupid IDE automation :D